### PR TITLE
move getAppConfigGroups to go

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

**Comment by laverya** [comment]
todo: dead code removals

File: kotsadm/pkg/handlers/config.go [hunk 1]
```diff
@@ -252,6 +258,94 @@ func LiveAppConfig(w http.ResponseWriter, r *http.Request) {
...
+	}
+
+	renderedConfig, err := kotsconfig.TemplateConfigObjects(kotsKinds.Config, configValues, appLicense, template.LocalRegistry{})
+	if err != nil {
+		currentAppConfigResponse.Error = "failed to render templates"
+		JSON(w, http.StatusInternalServerError, currentAppConfigResponse)
+		return
+	}
+
+	JSON(w, http.StatusOK, LiveAppConfigResponse{Success: true, ConfigGroups: renderedConfig.Spec.Groups})
```
Comment (salahalsaleh) [inline kotsadm/pkg/handlers/config.go:346]:
```suggestion
	JSON(w, http.StatusOK, currentAppConfigResponse)
```

Comment (salahalsaleh) [reply kotsadm/pkg/handlers/config.go:346]: and probably wanna do something like this before this line
```
currentAppConfigResponse.Success = true before
```

Comment (laverya) [reply kotsadm/pkg/handlers/config.go:346]: whoops
I'll fix that

**Pull request merged**